localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp

```diff
@@ -117,6 +103,9 @@ class EKFLocalizer : public rclcpp::Node
   EKFLocalizer(const std::string & node_name, const rclcpp::NodeOptions & options);
 
 private:
+  using PoseWithCovarianceStamped = geometry_msgs::msg::PoseWithCovarianceStamped;
```

Comment:
This is just a minor comment, but maybe we can incorporate this literal in another PR, since currently this literal is used only in two parts (and within this script and not in .cpp)

Reply:
Fixed in [109ffa7](https://github.com/autowarefoundation/autoware.universe/pull/2656/commits/109ffa77c0dd8523f1cfad0f45b88874b4e62238)